File: src/main/java/pe/edu/upeu/sys_api_restaurant/entity/LugarAtencion.java
package pe.edu.upeu.sys_api_restaurant.entity;

import jakarta.persistence.*;
import lombok.*;

@Entity
@Table(name = "LugarAtencion")
@Getter
@Setter
@NoArgsConstructor
@AllArgsConstructor
@Builder
public class LugarAtencion {

    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Integer idLugar;

    @Column(nullable = false, length = 80)
    private String nombreLugar;

    @Column(nullable = false, length = 20)
    private String tipoLugar;

    @Column(length = 200)
    private String direccion;

    private Integer capacidadMaxima;

    @Column(nullable = false)
    private Boolean estadoLugar;

    @Column(length = 255)
    private String observacion;
}

File: frontend/apprestaurant/src/app/core/models/lugar-atencion.ts
export interface LugarAtencion {
  idLugar?: number;
  nombreLugar: string;
  tipoLugar: string;
  direccion?: string;
  capacidadMaxima?: number;
  estadoLugar: boolean;
  observacion?: string;
}

export interface QRLugar {
  idQR?: number;
  codigoQR: string;
  urlQR?: string;
  estadoQR: boolean;
  lugarAtencion?: LugarAtencion;
}
